api/analytics/analytic_report.py

```diff
@@ -54,7 +54,6 @@ def get_least_used_rooms(self, all_rooms_summary_df):
         '''
         Get analytics for the 10 least used rooms
         '''
-        # import pdb; pdb.set_trace()
```

Comment:
Remove commented out code from the source control

Reply:
This comment was merged in a previous PR, I was removing it from the source code